state-chain/pallets/cf-environment/src/mock.rs

```diff
@@ -175,6 +176,13 @@ impl GetBitcoinFeeInfo for MockBitcoinFeeInfo {
 	}
 }
 
+pub struct MockBitcoinKeyProvider;
+impl KeyProvider<BitcoinCrypto> for MockBitcoinKeyProvider {
+	fn active_epoch_key() -> Option<EpochKey<AggKey>> {
+		CurrentBitcoinKey::get()
+	}
+}
```

Comment:
We already have a generic mock for `KeyProvider`, prefer to use that one (cf_traits::mocks::key_provider).